src/lib/common/globals.h

```diff
@@ -96,6 +96,7 @@
 #define DATE_EXPIRES    "dateExpires"
 #define ALTERATION_TYPE "alterationType"
 #define ALL_ATTRS       "*"
+#define SERVICE_PATH    "servicePath"
```

Comment:
```suggestion
#define SERVICE_PATH    "servicePath"
#define ALL_ATTRS       "*"
```

Better to have the "wildcard" as the last case.

Reply:
Resolve in c3eec37